pkg/util/modules/modules.go

```diff
@@ -57,6 +74,11 @@ func (m *Manager) InitModuleServices(target string) (map[string]services.Service
 	if _, ok := m.modules[target]; !ok {
 		return nil, fmt.Errorf("unrecognised module name: %s", target)
 	}
+
+	if !m.modules[target].option.Public {
```

Comment:
I think these are two separate concerns: whether module is public (visible in -modules) and whether it is runnable (can be selected for -target). There can be modules that are runnable, but not public. (for testing, during development)

I suggest we remove this check for now. We can introduce second option to make module "runnable" later.

Reply:
Make sense. I will update :)